html-forms/index.html

```diff
@@ -0,0 +1,42 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <title>HTML Forms</title>
+  </head>
+  <body>
+    <h1>Coding Form!</h1>
+    <form>
+      <label for="email">Email</label>
+      <div>
+        <input type="email" id="email">
+      </div>
+      <label for="dropdown">Why is coding cool?</label>
+      <div>
+        <select>
+          <option value="select">Select an option</option>
+          <option value="interesting">Because it's interesting</option>
+          <option value="fun">Because it's fun</option>
+          <option value="powerful">Because it's powerful</option>
+        </select>
+      </div>
+      <label for="radio">Favorite Language?(choose one)</label>
+      <div>
+        <input type="radio" name="language">
+        <label for="html" id="html">HTML</label>
+        <input type="radio" name="language">
+        <label for="css" id="CSS">CSS</label>
```

Comment:
Same thing here. Make sure the values match as well, capitalization needs to be consistent